monocle/web_utils.py

```diff
@@ -105,6 +105,9 @@ def sighting_to_marker(pokemon, names=POKEMON, moves=MOVES, damage=DAMAGE):
         marker['move2'] = moves[move2]
         marker['damage1'] = damage[move1]
         marker['damage2'] = damage[move2]
+    if pokemon.cp:
```

Comment:
it you add the cp column it should be filled with `null` or `0`, so I'd guess this is not needed as it would only be a "if there still are legacy pokemon" which should disapear after 30-60 mins